modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/PermissionUtil.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.admin.user.internal.dto.v1_0.util;
+
+import com.liferay.portal.kernel.service.PermissionService;
+import com.liferay.portal.kernel.service.ResourceActionLocalService;
+import com.liferay.portal.vulcan.permission.Permission;
+
+import java.util.Collection;
+
+/**
+ * @author Crescenzo Rega
+ */
+public class PermissionUtil {
+
+	public static Permission[] toPermissions(
+			long companyId, long groupId, long id, String name,
```

Comment:
can you please check if the groupId should go before the companyId?

Reply:
usually in the methods that are not of the service the alphabetical order is respected